compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/LexerTerminals.java

```diff
@@ -23,113 +23,114 @@
-    public static final String RESOURCE = "resource";
-    public static final String LISTENER = "listener";
-    public static final String CONST = "const";
-    public static final String FINAL = "final";
-    public static final String TYPEOF = "typeof";
-    public static final String IS = "is";
-    public static final String NULL = "null";
-    public static final String LOCK = "lock";
-    public static final String ANNOTATION = "annotation";
-    public static final String SOURCE = "source";
-    public static final String WORKER = "worker";
-    public static final String PARAMETER = "parameter";
-    public static final String FIELD = "field";
-    public static final String ISOLATED = "isolated";
-    public static final String XMLNS = "xmlns";
-    public static final String FORK = "fork";
-    public static final String TRAP = "trap";
-    public static final String IN = "in";
-    public static final String FOREACH = "foreach";
-    public static final String TABLE = "table";
-    public static final String KEY = "key";
-    public static final String ERROR = "error";
-    public static final String LET = "let";
-    public static final String STREAM = "stream";
-    public static final String NEW = "new";
-    public static final String READONLY = "readonly";
-    public static final String DISTINCT = "distinct";
-    public static final String FROM = "from";
-    public static final String WHERE = "where";
-    public static final String SELECT = "select";
-    public static final String START = "start";
-    public static final String FLUSH = "flush";
-    public static final String DEFAULT = "default";
-    public static final String WAIT = "wait";
-    public static final String DO = "do";
-    public static final String TRANSACTION = "transaction";
-    public static final String TRANSACTIONAL = "transactional";
-    public static final String COMMIT = "commit";
-    public static final String RETRY = "retry";
-    public static final String ROLLBACK = "rollback";
-    public static final String ENUM = "enum";
-    public static final String BASE16 = "base16";
-    public static final String BASE64 = "base64";
-    public static final String MATCH = "match";
-    public static final String CONFLICT = "conflict";
-    public static final String LIMIT = "limit";
-    public static final String JOIN = "join";
-    public static final String OUTER = "outer";
-    public static final String EQUALS = "equals";
-    public static final String ORDER = "order";
-    public static final String BY = "by";
-    public static final String ASCENDING = "ascending";
-    public static final String DESCENDING = "descending";
-    public static final String CLASS = "class";
-    public static final String CONFIGURABLE = "configurable";
+    public static final String PUBLIC = Configurations.getInstance().getProperty("public");
+    public static final String PRIVATE = Configurations.getInstance().getProperty("private");
```

Comment:
If the property mapping is missing; shouldn't we have defaults?

Reply:
Made this change (https://github.com/ballerina-platform/ballerina-lang/commit/84391b03d60675b3c2744c4c0fc346c9c5bc2d74)